scripts/azure_deployment/arm_aci.py

```diff
@@ -414,14 +399,26 @@ def make_aci_deployment(args: Namespace) -> Deployment:
         if args.generate_security_policy:
             with tempfile.TemporaryDirectory() as tmpdirname:
                 arm_template_path = f"{tmpdirname}/arm_template.json"
+                output_policy_path = f"{tmpdirname}/security_policy"
                 with open(arm_template_path, "w") as f:
                     json.dump(arm_template, f)
                 # sudo is necessary for docker to avoid error "The current user does not have permission".
                 # The recommended solution is 'sudo usermod -aG docker', but it requires re-login.
                 # https://docs.docker.com/engine/install/linux-postinstall/
                 # We use sudo instead as a workaround.
                 completed_process = subprocess.run(
-                    ["sudo", "az", "confcom", "acipolicygen", "-a", arm_template_path],
+                    [
+                        "sudo",
+                        "az",
+                        "confcom",
+                        "acipolicygen",
+                        "-a",
+                        arm_template_path,
+                        "--pretty-print",
+                        "--outraw-pretty-print",
+                        "--save-to-file",
+                        output_policy_path,
```

Comment:
It's getting error in the CI
```
ERROR: unrecognized arguments: --pretty-print
```
I only tried `--pretty-print --outraw-pretty-print` without `--save-to-file`. So we might have to remove the option and get the result from STDOUT.